conf/ohsnap.yaml.template

```diff
@@ -0,0 +1,7 @@
+OHSNAP:
+  HOST: "replace by ohsnap repo host url"
+  REQUEST_RETRY:
```

Comment:
Could you remove a old `OHSNAP_REPO_HOST` from template and config validators?

Reply:
done